tests/labhub_test.py

```diff
@@ -292,5 +292,7 @@ def test_invite_me(self):
         plugins.labhub.os.environ['GH_TOKEN'] = 'patched?'
         testbot.assertCommand('!invite me',
                               'We\'ve just sent you an invite')
+        testbot.assertCommand('!hey there invite me',
+                              '', timeout=100)
```

Comment:
Change this to `'We\'ve just sent you an invite'` and not an empty string `''` The tests should fail after this change.

Reply:
@meetmangukiya No, corobo should not be triggered by this, there should be no text before and after the command. So, there should be no output here.

Reply:
Yeah, and hence, the tests will fail :wink:

Reply:
so what should we do about them? add an exception in .coafile to avoid failure?